tests/unit/neptune/new/test_handler.py

```diff
@@ -236,14 +236,15 @@ def test_append_complex_input(self):
                 {
                     "key-a": {"aa": 11, "ab": 22},
                     "key-b": {"ba": 33, "bb": 44},
-                }
+                },
+                step=1,
             )
             assert exp["train"]["dictOfDicts"]["key-a"]["aa"].fetch_last() == 11
             assert exp["train"]["dictOfDicts"]["key-a"]["ab"].fetch_last() == 22
             assert exp["train"]["dictOfDicts"]["key-b"]["ba"].fetch_last() == 33
             assert exp["train"]["dictOfDicts"]["key-b"]["bb"].fetch_last() == 44
 
-    @pytest.mark.xfail(reason="File logging disabled", strict=True, raises=NeptuneUnsupportedFunctionalityException)
+    @pytest.mark.xfail(reason="Doesn't work with step enforcement", strict=True, raises=TypeError)
```

Comment:
wait we didn't enable any new features. Why `NeptuneUnsupportedFunctionalityException` is not raised in this test?